plugins/kjob.yaml

```
@@ -0,0 +1,45 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: kjob
+spec:
+  version: v0.1.0
+  shortDescription: Tool for CLI-loving ML researchers allowing them to run AI/ML jobs based on the pre-defined templates.
```

```suggestion
  shortDescription: Run AI/ML jobs based on the pre-defined templates
```
we truncate at 55 chars